data/json/items/comestibles/veggy_dishes.json

```diff
@@ -533,6 +533,24 @@
     "fun": 3,
     "vitamins": [ [ "vitC", 9 ], [ "calcium", 13 ], [ "iron", 8 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "potato_boiled",
+    "name": { "str": "boiled potato", "str_pl": "boiled potatoes" },
+    "weight": "150 g",
+    "flags": [ "EATEN_HOT", "NUTRIENT_OVERRIDE" ],
```

Comment:
Why NUTRIENT_OVERRIDE?

Reply:
Using this [cheat sheet](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_FLAGS.md#flags-2) , the description for the flag "NUTRIENT_OVERRIDE" was a bit vague, so I have no idea what it's for, but I saw that baked potatoes used it so I assumed it was appropriate to put it there?

Reply:
It looks like the baked potato has that because it was added before the `RAW` flag existed (https://github.com/CleverRaven/Cataclysm-DDA/pull/34989). I think you should remove it here.

Crafted foods will inherit vitamins and nutrients from the items they are crafted from, unless they have this flag. Because the crafting recipe involves cooking, the `RAW` flag from the potato will not be applied, and so the calories in the boiled potato will be appropriately higher than the raw potato.

Reply:
Thanks for clarifying. I just made the changes.